src/viztracer/__main__.py

```diff
@@ -3,6 +3,5 @@
 
 from .main import main
 
-
```

Comment:
The style of blank lines between import and the code is inconsistent in CPython repo itself. However, I'd like to keep 2 lines, so let's revert all the line reductions(except for the line between `__all__` and import, I can consider `__all__` as part of import)